servers:
  image_segmentation:
    service: /butia_vision/seg/image_segmentation